.github/workflows/push.yml

```diff
@@ -23,33 +23,21 @@ jobs:
   publish:
     needs: [pack, test]
     runs-on: ubuntu-latest
+
     steps:
     - name: Setup .NET
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: ${{ env.DOTNET_VERSION }}
         source-url: ${{ env.AZURE_ARTIFACTS_FEED_URL }}
```

Comment:
should this AZURE be updated?

Reply:
I can change back, but it was using github repo

Reply:
nonono, i am just asking, still checking.

Reply:
looking it, doesnt look like we are using the repo at all in this `yml` file